deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -3,7 +3,8 @@ receivers:
   jaeger:
     protocols:
       grpc:
-  zipkin: {}
+  zipkin:
+    endpoint: "0.0.0.0:9411"
```

Comment:
I think we could check if hostname name from `values.yaml` will work in this place.

Reply:
I could use `{{ printf "%s-otelcol" (include "sumologic.fullname" .) }}`, but it will be easy to omit while updating collector name :)